### ISSUE TITLE
Log spam

### ISSUE THREAD

**Issue by michaelsproul** [sigp/lighthouse#1463]
## Description

In the early stages of the Medalla network my node recorded two massive influxes of dropped logs:

```
Aug 02 16:40:51.241 ERRO slog-async: logger dropped messages due to channel overflow, count: 124863, service: beacon
...
Aug 04 23:37:25.029 ERRO slog-async: logger dropped messages due to channel overflow, count: 20748
```

The first one occurred pre-genesis, and seems like a ridiculous number (124863 log entries!), possibly a logging bug?

The second one was post-genesis and could be "just" invalid attestations, but 20k messages still seems like a lot.

This is running with `--debug-level=debug`

## Version

```
Lighthouse Lighthouse/v0.2.0/f26adc0a
BLS Library: blst
```

**Comment by paulhauner** [comment]
I believe this was addressed in #1464

**Comment by michaelsproul** [comment]
I'd kind of still like to know what produces 124863 logs
